src/main/bin/postman.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Bash script to run a Postman collection with a dynamically calculated API_URL
+
+# TODO: Calculate or fetch the API_URL
+# Replace this placeholder with the actual method to determine API_URL.
+API_URL="http://localhost:8082/api/v1"  # Placeholder URL; update as needed.
+# https://api-evsrest.nci.nih.gov - Prod API_URL
+# https://api-test-evsrest.nci.nih.gov - Test API_URL
+# https://api-qa-evsrest.nci.nih.gov - QA API_URL
+
+# Ensure API_URL is set in the environment
+if [ -z "$API_URL" ]; then
+    echo "Error: API_URL is not set. Please export API_URL before running this script."
+    exit 1
+fi
+
+# Default value for the collection name placeholder
+COLLECTION_NAME="${1:-ncit}"
+
+# Construct the collection file name with the specified or default collection name
+COLLECTION_FILE="EVSRESTAPI_Postman_${COLLECTION_NAME}_Demo.postman_collection.json"
+
+# Verify if newman is installed
+if ! command -v newman &> /dev/null; then
+    echo "Newman is not installed. Attempting to install it locally..."
+    # Try installing Newman locally
+    npm install newman
+    if [ $? -ne 0 ]; then
+        echo "Error: Newman installation failed. Please install it manually."
+        exit 2
+    fi
+fi
```

Comment:
I wasn't able to get to the bash script to run locally, but this was due to `newman` not installing correctly on my machine. It looks like it installs, but when `newman` command is called, it's not getting recognized. As a valid check. We should add a check to make sure that this section that installs the command was successful by checking `newman` is recognized by the terminal. Otherwise we get this error, which doesn't indicate to the user why the install failed
```
./postman.sh: line 37: newman: command not found
Error: Postman collection failed with status code 127.
```

Reply:
Alright I added another check there, but I couldn't properly test it since the script worked fine on my mac including the newman install.